lib/tidewave/mcp/server.ex

```diff
@@ -15,12 +15,16 @@ defmodule Tidewave.MCP.Server do
     tools = raw_tools()
     dispatch_map = Map.new(tools, fn tool -> {tool.name, tool.callback} end)
 
-    :persistent_term.put({__MODULE__, :tools_and_dispatch}, {tools, dispatch_map})
+    # TODO: switch back to persistent_term when we don't support OTP 27 any more
+    # :persistent_term.put({__MODULE__, :tools_and_dispatch}, {tools, dispatch_map})
+    Tidewave.Tools.store_tools({tools, dispatch_map})
```

Comment:
This is called on application boot, so it's never concurrent. We could just create the ETS table here and insert the entry here, without the extra GenServer.

Reply:
Right, I just wasn't sure how clean it would be to have whichever process calls Application.start own the table?

Reply:
Ah, yes that’s alright 👍🏻